gateway/ln-gateway/src/gateway_module_v2/send_sm.rs

```diff
@@ -170,7 +170,8 @@ impl SendStateMachine {
             lightning_context
                 .lnrpc
                 .pay_private(
-                    PrunedInvoice::try_from(invoice).expect("Invoice has amount"),
+                    // todo support override amount
+                    PrunedInvoice::try_from(&invoice, None).expect("Invoice has amount"),
```

Comment:
I'm wondering why v2 does the conversion on the GW side, leaking the description to the gateway operator. @joschisan

Reply:
Its a hack until pruned invoices are supported on cln as well.